spec/game_presenter_spec.rb

```diff
@@ -0,0 +1,41 @@
+require_relative '../lib/game_presenter/game_presenter'
+require_relative '../lib/game_presenter/test_io'
+require_relative '../lib/game_presenter/console_io'
+
+describe 'Game Presenter' do
+    context 'Initialization' do
+        game_presenter = GamePresenter.new(
+            presenter: ConsoleIO.new()
+        )
+        it 'sets presenter' do
+            expect(game_presenter.presenter.is_a?(ConsoleIO)).to eq true
+        end
+    end
+
+    context 'get_input with GameIO' do
+        game_presenter = GamePresenter.new(
+            presenter: TestIO.new()
+        )
+        it 'gets input through presenter' do
```

Comment:
I think the presenter should only be in charge of presenting the board, not getting input.

Reply:
Is it a problem of the name? Should I separate input output, or just change the name to GameIO?

Reply:
It might just be the name